### DISCUSSION TITLE
[Feature] Switch between thumbnail display and facebox display in People groups

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#6013] [Feature Request]
### The feature

It would be awesome to be able to show only the faces (bounding box), rather than the entire pic thumbnail, of a given person.
Like a switch between different display modes.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by shumancw** [comment]
My input:
Issue: Especially in family photos where there can be a lot of similarities between people, there are many time when the photos presented contain two people that look similar and the model chooses the wrong person.

Example: 10 photos are presented containing a group of people. The person I think is the person identified in all 10 photos is actually only identified correctly in 8 of the photos. In 2 of the photos presented it actually identified a cousin, for example, as being the same as the person in the other 8 photos.

Feature: Give the ability for the 10 pictures of the unnamed person to show as thumbnails or facebox in version of the person identified. Doing this allow for a user to quickly confirm that 8 of the photos belong to one person and 2 to another.

**Comment by badbob001** [comment]
I NEED this feature.

I just had to reset all my face data because there were incorrect faces in the grouping but it would take FOREVER to removed them one photo at a time. The issue is that is very hard to find incorrect faces under each person grouping since it's lacking a face-only view. The old google picasa had this and made people management much easier.
